src/pages/Faucet/index.tsx

```diff
@@ -67,27 +70,32 @@ export const FaucetPage = () => {
           height="300"
         />
         <div className="flex flex-col gap-5 w-full md:w-56">
-          {Object.keys(StaticConfig.tokenAddresses).map(key => (
+          {Object.keys(config?.tokenAddresses || {}).map(key => (
```

Comment:
consider using Object.entries instead if we going to need both key and value

Reply:
refactored entirely